src/parser/jobs/nin/ShadowFang.js

```diff
@@ -74,14 +75,19 @@ export default class ShadowFang extends Module {
 		}))
 
 		// Suggestion for DoT clipping
-		const severity = this.getDotClippingSeverity()
-		if (severity) {
-			this.suggestions.add(new Suggestion({
+		const clipping = this.getDotClippingAmount()
+		if (clipping >= CLIP_SECS_PER_MIN_THRESHOLD) {
```

Comment:
You can remove this `if` - `TieredSuggestion` will automatically `SEVERITY.IGNORE` if it doesn't meet the minimum tier.

Reply:
Ooh, nice. I'll update that.